hack/common.sh

```diff
@@ -51,6 +51,8 @@ KUBEVIRT_RELEASE=${KUBEVIRT_RELEASE:-false}
 OPERATOR_MANIFEST_PATH=$MANIFESTS_OUT_DIR/release/kubevirt-operator.yaml
 TESTING_MANIFEST_PATH=$MANIFESTS_OUT_DIR/testing
 KUBEVIRT_CRI="$(determine_cri_bin)"
+# The image list is sync with the images in container_bundle "build-other-images" in BUILD.bazel
```

Comment:
@zhlhahaha Hi, how did you manage to get the digest for the utility images? IIUC these images were pushed as a bundle using `docker_push` which doesn't produce a digest

Reply:
Yes, they do not generate digests. Are there any places where we require utility image digests?

Reply:
> @zhlhahaha Hi, how did you manage to get the digest for the utility images? IIUC these images were pushed as a bundle using `docker_push` which doesn't produce a digest

Hi @enp0s3 , I have updated the PR and using digest rather than tag for pushing multi-arch manifest. Please take a look when you have time.